modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/ConfigurablePublishWorkflowOperationHandler.java

```diff
@@ -124,6 +136,36 @@ protected DownloadDistributionService getDistributionService() {
     return distributionService;
   }
 
+  private void addMetadataCatalogsToTemplateData(Map<String, Object> values, MediaPackage mp) {
+    if (! mp.hasCatalogs()) {
+      logger.info("No metdata catalogs found.");
+      return;
+    }
+    if (values == null) values = new HashMap<>();
+
+    for (Catalog catalog : mp.getCatalogs()) {
+      if (catalog.getFlavor() != null) {
+        String mainflavor = catalog.getFlavor().getType();
+        String subflavor = catalog.getFlavor().getSubtype();
+        Map<String, Object> metadata = new HashMap<>();
+        try {
+          DublinCoreCatalog dc = DublinCoreUtil.loadDublinCore(workspace, catalog);
+          for (Map.Entry<EName, List<DublinCoreValue>> entry : dc.getValues().entrySet()) {
+            String value = entry.getValue().get(0).getValue();
```

Comment:
This will discard all but the first value in case you have e.g. multiple presenters. That's generally okay but is not what's documented. either adjust this or the documentation.